README.md

```diff
@@ -147,7 +147,7 @@ Example usage:
 $ rbenv install --patch 1.9.3-p429 < /path/to/ruby.patch
 
 # applying a patch from HTTP
-$ rbenv install --patch 1.9.3-p429 < <(curl -sSL http://git.io/ruby.patch)
+$ rbenv install --patch 1.9.3-p429 < <(curl -sSL https://git.io/ruby.patch)
```

Comment:
This change wasn't necessary. The URL is for example only, and doesn't even resolve to anything (`ruby.patch` is a 404).